File: cvsi/cvsi-core/src/main/java/com/winify/cvsi/core/dto/ImageDto.java
package com.winify.cvsi.core.dto;

import com.winify.cvsi.core.dto.error.ServerResponseStatus;
import com.winify.cvsi.db.model.enums.ImageType;

import java.io.Serializable;

public class ImageDto extends ServerResponseStatus implements Serializable {
    private ImageType imageType;
    private String image;
    private Long createdDate;

    public ImageType getImageType() {
        return imageType;
    }

    public void setImageType(ImageType imageType) {
        this.imageType = imageType;
    }

    public String getImage() {
        return image;
    }

    public void setImage(String image) {
        this.image = image;
    }

    public Long getCreatedDate() {
        return createdDate;
    }

    public void setCreatedDate(Long createdDate) {
        this.createdDate = createdDate;
    }
}

File: Mobile/app/CVSI/app/src/main/java/android/cvsi/models/ImageDto.kt
package android.cvsi.models

import android.cvsi.models.error.ServerResponseStatus
import com.bumptech.glide.load.resource.bitmap.ImageHeaderParser
import java.io.Serializable

class ImageDto : ServerResponseStatus(), Serializable {
    var imageType: ImageHeaderParser.ImageType? = null
    var image: String? = null
    var createdDate: Long? = null
}
